ksql-test-util/src/main/java/io/confluent/ksql/test/util/ConsumerTestUtil.java

```diff
@@ -151,6 +151,48 @@ private ConsumerTestUtil() {
     return acquired;
   }
 
+  /**
+   * Get all records available to the supplied {@code consumer}.
+   *
+   * @param consumer the consumer to use.
+   * @param <K> the Kafka record key type.
+   * @param <V> the Kafka record value type.
+   * @return the consumed messages.
+   */
+  public static <K, V> List<ConsumerRecord<K, V>> getAvailableRecords(
+      final Consumer<K, V> consumer
+  ) {
+    return getAvailableRecords(consumer, DEFAULT_VERIFY_TIMEOUT);
+  }
+
+  /**
+   * Get all records available to the supplied {@code consumer}.
+   *
+   * @param consumer the consumer to use.
+   * @param timeout how long to wait for the records to be available.
+   * @param <K> the Kafka record key type.
+   * @param <V> the Kafka record value type.
+   * @return the consumed records.
+   */
+  public static <K, V> List<ConsumerRecord<K, V>> getAvailableRecords(
```

Comment:
Do we need both APIs here? I'd just hardcode the default timeout until we need to specify it.

Reply:
I've removed this now since I realized this isn't necessary to get the queryId.